### DISCUSSION TITLE
Console / request log

### DISCUSSION THREAD

**Discussion by phatdaddy-xda** [warmuuh/milkman#184] [Q&A]
Postman has decided to move to a SaaS model for delivering all their functionality and has moved away from being able to store local copies of a collection, forcing users to adopt their online workspace feature for collections, which is against company policy.  We primarily use Postman to configure third-party software, but it requires a LOT of REST API calls to do the configuration, so our collection is quite large.  We also need to remain backward-compatible with a vendor who uses Postman to send us collections occasionally. 

It looks like Milkman is a good alternative to Postman.  However, despite importing our Postman collections successfully, I'm having trouble getting requests--particularly the one that gets a token--to work correctly.  While I expect I can convert to using the MM authentication plugin, I don't see anything akin to the console in Postman so I can't easily diagnose issues with individual requests in our collection.  How can I see the actual requests that Milkman is sending to the API?

Thanks!

**Comment by warmuuh** [comment]
there is a setting for enabling debug output, which shows you details of what got actually sent to the server
<img width="492" alt="image" src="https://github.com/warmuuh/milkman/assets/1331628/d4909d06-8e67-4b05-a60e-2dff7f4b9c89">
